vizro-core/docs/pages/user_guides/navigation.md

```diff
@@ -0,0 +1,138 @@
+# How to use the navigation panel
+
+This guide shows you how to use and customize the left navigation panel in your dashboard.
+
+The [`Dashboard`][vizro.models.Dashboard] model also accepts a `navigation` argument, where you can enter the [`Navigation`][vizro.models.Navigation] model. A [`Navigation`][vizro.models.Navigation] in your dashboard facilitates grouping and navigating through your pages.
+
+## Using the default navigation
+
+By default, a navigation panel with an accordion item per page and a default title of `SELECT PAGE` is added to your dashboard. To leverage the navigation bar, you do not need to configure anything on top.
+
+!!! example "Default navigation"
+    === "app.py"
+        ```py
+        from vizro import Vizro
+        import vizro.plotly.express as px
+        import vizro.models as vm
+
+        iris = px.data.iris()
+
+        page_1 = vm.Page(
+            title="My first page",
+            components=[
+                vm.Graph(id="scatter_chart", figure=px.scatter(iris, x="sepal_length", y="petal_width", color="species")),
+            ],
+        )
+
+        dashboard = vm.Dashboard(pages=[page_1])
+
+        Vizro().build(dashboard).run()
+        ```
+    === "app.yaml"
+        ```yaml
+        # Still requires a .py to register data connector in Data Manager and parse yaml configuration - see from_yaml example
```

Comment:
Can we split this over two lines (like I think other yaml examples), this avoids the scroll